pallet/src/lib.rs

```diff
@@ -163,8 +176,13 @@ pub mod pallet {
             <SessionPeriod<T>>::put(&self.session_period);
             <MillisecsPerBlock<T>>::put(&self.millisecs_per_block);
             <UnitCreationDelay<T>>::put(&self.unit_creation_delay);
+
+            // TODO
             <Validators<T>>::put(Some(&self.validators));
             <SessionForValidatorsChange<T>>::put(Some(0));
+
+            // <Validators<T>>::put(None::<Vec<T::AccountId>>);
+            // <SessionForValidatorsChange<T>>::put(None::<u32>);
```

Comment:
What is the point of these commented lines? are we going to uncomment them soon?

Reply:
commited by mistake, reverted